### ISSUE TITLE
Impact of using different cellbender parameters across samples on downstream analysis

### ISSUE THREAD

**Issue by diego-qt** [broadinstitute/CellBender#271]
Hi @sjfleming and team!

First of all, thanks for creating such a useful tool. While analyzing data that we generated along with some external public datasets, a question came to mind. Are there any cellbender parameters that, if used differently across samples, could bias downstream analysis? From what I understand from the paper, cellbender does an excellent job at correcting spurious differential gene expression caused by background noise. However, I’m curious if using different parameters between samples could introduce some biases/differences. My logical reasoning tells me that adjusting parameters such as the number of epochs and learning rate to achieve near-optimal training in each sample wouldn’t matter due to the independent nature of each sample. On the other hand, I suspect that using a different false positive rate (FPR) to remove varying amounts of noise could lead to differences across samples introduced by the background removal procedure. Is my understanding correct?

Thanks again!
